articles/container-service/container-service-kubernetes-ui.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Using the Kubernetes Web UI in Azure Container Service | Microsoft Docs
+description: Using the Kubernetes Web UI in Azure Container Service
+services: container-service
+documentationcenter: ''
+author: bburns
+manager: timlt
+editor: ''
+tags: acs, azure-container-service, kubernetes
+keywords: ''
+
+ms.assetid: what-goes-here?
+ms.service: container-service
+ms.devlang: na
+ms.topic: article
+ms.tgt_pltfrm: na
+ms.workload: na
+ms.date: 12/09/2016
+ms.author: bburns
+
+---
+
+# Microsoft Azure Container Service Engine - Using the Kubernetes Web UI
+
+## Prerequisites
+This walkthrough assumes that you have [created a Kubernetes cluster using Azure Container Service](container-service-kubernetes-walkthrough.md).
+
+It also assumes that you have the azure cli and kubectl tools installed.
```

Comment:
Azure CLI 2.0 (Preview) is the current name. As opposed to the node version, Azure CLI 1.0